draft-ietf-rats-architecture.md

```diff
@@ -106,8 +106,9 @@ such as roles, device composition, topological models, and appraisal is vital fo
 semantic interoperability across solutions and platforms involving multiple vendors and providers.
 
 Amongst other things, this document is about trust and trustworthiness.
-Trust is a decision being made. Trustworthiness is a quality that is
-assessed via evidence created. This is a subtle difference and being
+The decision to trust spans beyond technical assertions to focus on trustworthiness as a quality of the computing environment.
+Trustworthiness is a quality that is
+assessed via evidence. This is a subtle difference and being
```

Comment:
Trust is a choice one makes about another system.   Trustworthiness is a quality about the other system that can be used in making one's decision to trust it or not.